editor/scene_tree_dock.h

```diff
@@ -135,6 +135,7 @@ class SceneTreeDock : public VBoxContainer {
 
 	EditorData *editor_data = nullptr;
 	EditorSelection *editor_selection = nullptr;
+	List<Node *> node_previous_selection;
```

Comment:
```suggestion
	LocalVector<Node *> node_previous_selection;
```
No reason to use a List.
Also remember to `reserve()` before adding elements.